code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -134,3 +134,9 @@
 	life = "Unknown. No collected sample has been active long enough to determine lifespan."
 	notes = "Manually activated by the user."
 	function = "Allows the user to summon a box from a bluespace pocket located inside the implant. The exterior of this box is lined with experimental cloaking panels which render the box invisible to the naked eye."
+
+/datum/implant_fluff/shock
+	name = "Syndicate ARC-7 \"Power\" Bio-chip"
+	life = "5-12 months, depending on active use."
+	notes = "Manually activated by the user."
+	function = "An effective biochip designed to redirect large amounts of power in destructive arcs. Attempts to reduce the minimum power drain to allow for weapon and cell charging have failed so far."
```

Comment:
```suggestion
	function = "An effective biochip designed to redirect large amounts of power in destructive arcs. Attempts to reduce the minimum power drain, for weapon and cell charging, have failed so far."
```
Totally not required, but I think something like this flows a bit better.

Reply:
I like the original version better. The commas seem unnecessary - the sentence flows better without them.